test/integration/kind-cluster-config.yaml

```diff
@@ -0,0 +1,6 @@
+kind: Cluster
+apiVersion: kind.x-k8s.io/v1alpha4
+featureGates:
+  # due to the absence of scheduler, we need to enable the PodScheduling feature
+  # to be able to block the pods until the pre tasks/evals are finished
+  "PodSchedulingReadiness": true
```

Comment:
is this needed in 1.27.3 ? I thought gates where enabled by default there 😮

Reply:
You are right, it's not needed anymore